app/controllers/course/forum/forums_controller.rb

```diff
@@ -72,4 +73,9 @@ def load_forum
   def forum_params
     params.require(:forum).permit(:name, :description, :course_id)
   end
+
+  def add_forum_item_breadcrumb
+    add_breadcrumb @forum.name, course_forum_path(current_course, @forum) unless @forum.nil? ||
```

Comment:
I think topic should add its title and its forum's name together. Forum name can not be put in the abstract controller because in abstract controller we can not get `@forum` in every situations.